modules/catalog/src/main/scala/lucuma/catalog/CatalogAdapter.scala

```diff
@@ -256,58 +270,66 @@ object CatalogAdapter {
     override def parseName(entries: Map[FieldId, String]): Option[String] =
       super.parseName(entries).map(_.stripPrefix("NAME "))
 
-    override def ignoreMagnitudeField(v: FieldId): Boolean =
+    override def ignoreBrightnessValueField(v: FieldId): Boolean =
       !v.id.value.toLowerCase.startsWith("flux") ||
         v.id.value.matches(errorFluxIDExtra) ||
         v.id.value.matches(fluxIDExtra)
 
-    override def isMagnitudeSystemField(v: (FieldId, String)): Boolean =
+    override def isBrightnessUnitsField(v: (FieldId, String)): Boolean =
       v._1.id.value.toLowerCase.startsWith("flux_system")
 
-    // Simbad has a few special cases to map sloan magnitudes
-    def findBand(id: FieldId): Option[MagnitudeBand] =
+    // Simbad has a few special cases to map sloan band brightnesses
+    def findBand(id: FieldId): Option[Band] =
       (id.id.value, id.ucd) match {
         case (magSystemID(b), _) => findBand(b)
         case (errorFluxID(b), _) => findBand(b)
         case (fluxID(b), _)      => findBand(b)
         case _                   => none
       }
 
-    // Simbad doesn't put the band in the ucd for magnitude errors
-    override def isMagnitudeErrorField(v: (FieldId, String)): Boolean =
+    // Simbad doesn't put the band in the ucd for  brightnesses errors
+    override def isBrightnessErrorField(v: (FieldId, String)): Boolean =
       v._1.ucd.includes(VoTableParser.UCD_MAG) &&
         v._1.ucd.includes(VoTableParser.STAT_ERR) &&
         errorFluxID.findFirstIn(v._1.id.value).isDefined &&
-        !ignoreMagnitudeField(v._1) &&
+        !ignoreBrightnessValueField(v._1) &&
         v._2.nonEmpty
 
-    protected def findBand(band: String): Option[MagnitudeBand] =
-      MagnitudeBand.all.find(_.shortName === band)
+    protected def findBand(band: String): Option[Band] =
+      Band.all.find(_.shortName === band)
 
-    override def fieldToBand(field: FieldId): Option[MagnitudeBand] =
-      if ((field.ucd.includes(VoTableParser.UCD_MAG) && !ignoreMagnitudeField(field)))
+    override def fieldToBand(field: FieldId): Option[Band] =
+      if ((field.ucd.includes(VoTableParser.UCD_MAG) && !ignoreBrightnessValueField(field)))
         findBand(field)
       else
         none
 
-    // Attempts to find the magnitude system for a band
-    override def parseMagnitudeSys(
+    private val integratedBrightnessUnits: Map[String, UnitType @@ Brightness[Integrated]] =
+      Map(
+        "Vega" -> implicitly[IsTaggedUnit[VegaMagnitude, Brightness[Integrated]]],
+        "AB"   -> implicitly[IsTaggedUnit[ABMagnitude, Brightness[Integrated]]]
```

Comment:
What about the other units? Or is that simbad only has AB/Vega?

Reply:
I based this on the field description for `FLUX_SYSTEM_*` which states `flux system A=AB V=Vega`. But now I see there's another field called `FLUX_UNIT_*` which is probably the one we want to use. However, I cannot find a list of possible values for it.

Reply:
Ok, `FLUX_UNIT_*` doesn't seem to be working. According to documentation (http://simbad.u-strasbg.fr/guide/sim-fscript.htx#VotableFields) it should return the flux units but it's returning the name of the band.

I think we should stick with `FLUX_SYSTEM_*` then. Also, despite its description it actually contains values `AB` and `Vega`.

Reply:
👍